### PR TITLE
Fix small bug when recopy style in order to compute the size of the component when it is hidden.

### PR DESCRIPTION
**Pull request by jDramaix**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
         <guava.version>18.0</guava.version>
         <gsss.version>1.0</gsss.version>
         <checkstyle.version>6.0</checkstyle.version>
-        <arcbees-checkstyle.version>1.1-SNAPSHOT</arcbees-checkstyle.version>
+        <arcbees-checkstyle.version>1.0</arcbees-checkstyle.version>
```
Comment (christiangoudreau) [inline pom.xml]: Don't you need to use 1.1-SNAPSHOT?

**Comment by christiangoudreau** [comment]
@meriouma There's problem with our checkstyle project?

**Comment by christiangoudreau** [comment]
ChosenIT.java:54: Line matches the illegal pattern 'check that a space is left after a colon on an assembled error message'.

**Comment by jDramaix** [comment]
This rule should be removed or at least not considered as Error. I've already seen a lot of cases where this rule failed but the line was valid

File: pom.xml (hunk 1 above)
Comment (jDramaix) [inline pom.xml]: Nope, they aren't any changes in the snapshot we need for the project.

**Comment by christiangoudreau** [comment]
I agree and though SP had removed it already, isn't that part of 1.1-SNAPSHOT @meriouma ?

**Comment by meriouma** [comment]
Yeah you're right SP removed it from build-tools. I'll release the version 1.1 this week

**Comment by spg** [comment]
@jDramaix would that fix benefit from having a test?

**Comment by meriouma** [comment]
LGTM

File: pom.xml (hunk 1 above)
Comment (meriouma) [inline pom.xml]: You should be able to switch back to 1.1. The rule is removed

**Comment by jDramaix** [comment]
@spg hard to test, the bug occurred during a very short interval on a temporary dom element

**Pull request merged**